### PR TITLE
btfhub: create pkgs for btfhub app

### PR DESCRIPTION
**Pull request by rafaeldtinoco**

### CODE REVIEW

**Comment by rafaeldtinoco** [comment]
Ooops forgot to bring back the tests, will do in another commit.

**Comment by rafaeldtinoco** [comment]
Alright, lets see what you think whenever you have some time. Cheers!

**Comment by rafaeldtinoco** [comment]
@brycekahle let me know if you are okay with this change whenever you can (no rush). No code changes besides exporting (from new pkgs) some functions and vars.

**Comment by rafaeldtinoco** [comment]
Hey @brycekahle do you think you will be able to review this this week ? I wanted to automate the kernel generations. Totally okay if you can't, I'll just go on with it and test all generations one by one. Thanks!

File: pkg/kernel/kernel.go [hunk 1]
```diff
@@ -1,28 +1,28 @@
-package main
+package kernel
 
 import (
 	"regexp"
 	"strconv"
 )
 
-type kernelVersion struct {
+type KernelVersion struct {
```
Comment (brycekahle) [inline pkg/kernel/kernel.go:8]: this stutters `kernel.KernelVersion`. Maybe name it just `Version`, so it becomes `kernel.Version`?

File: pkg/package/package.go [hunk 2]
```diff
@@ -1,27 +1,30 @@
-package main
+package pkg
```
Comment (brycekahle) [inline pkg/package/package.go:1]: Should the directory be named `pkg`, since you cannot name a package `package`?

File: pkg/repository/amazon.go [hunk 3]
```diff
@@ -1,41 +1,46 @@
-package main
+package repo
```
Comment (brycekahle) [inline pkg/repository/amazon.go:1]: I'd prefer if the package name matched the directory here too

**Review by brycekahle — DISMISSED** [review]
Some minor comments about package and type naming, but otherwise looks good.

**Comment by rafaeldtinoco** [comment]
Alright, I'll address the comments and merge it today, and automate the archive updates.

**Pull request merged**